.buildkite/pipeline.yml

```diff
@@ -1,6 +1,36 @@
 steps:
-  - commands:
+  - label: ":hammer: running tests"
+    commands:
       - "make ci"
-    name: ":hammer: build"
+        #- "make test" #currently no ct defined, causes build error
```

Comment:
Maybe fix the Makefile to not run tests since we don't have any?

Reply:
I was going to add a "fake" ct test so make test works but doesn't do anything. Then when someday we add tests it will "just work" theoretically